src/components/OSCALLoader.js

```diff
@@ -233,6 +263,11 @@ export default function OSCALLoader(props) {
     }
   }, [props.isRestMode]);
 
+  // Handle anchor link change in window url
+  useEffect(() => {
+    handleFragment();
+  }, [handleFragment]);
```

Comment:
Why not just have `handleFragment()`'s body here? I don't see where it gets used/passed anywhere else. And we can just change the triggers... Basically, why not just change the `useCallback` above to a `useEffect`?

Reply:
In an attempt to get rid of the linter warnings when using `window.location.hash` to aquire the hash over `useLocation()`, I had tried `useCallback()` but I remember receiving a warning saying I needed to use the function in conjunction with a `useEffect()`. I don't believe `useCallback()` is necessary with this problem solved, so I'll just shift it back to the way it was before.